tests/tests_lite/utilities/test_distributed.py

```diff
@@ -29,17 +19,11 @@ def _test_all_gather_uneven_tensors(rank, world_size, backend):
         assert (result[idx] == torch.ones_like(result[idx])).all()
 
 
-def _test_all_gather_uneven_tensors_multidim(rank, world_size, backend):
-    os.environ["MASTER_ADDR"] = "localhost"
-
-    if backend == "nccl":
-        device = torch.device("cuda", rank)
-        torch.cuda.set_device(device)
```

Comment:
This is ok to remove, because it is handled by the utility function spawn_launch -> setup_environment -> accelerator.setup_device -> torch.cuda.set_device